api/v1beta1/remotesecret_types.go

```diff
@@ -111,6 +113,14 @@ type TargetStatus struct {
 	// Error the optional error message if the deployment of either the secret or the service accounts failed.
 	// +optional
 	Error string `json:"error,omitempty"`
+	// Secret contains the status information about the linked secret in the target
+	Secret TargetSecretStatus `json:"secret"`
```

Comment:
Could we name this `deployedSecret` or something similar? Because we have already have status.secret and also status.targets[].expectedSecret